File: src/main/java/com/leijendary/spring/microservicetemplate/model/SampleTableTranslations.java
package com.leijendary.spring.microservicetemplate.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

import javax.persistence.Embeddable;

@Data
@EqualsAndHashCode(callSuper = true)
@Embeddable
public class SampleTableTranslations extends LocaleModel {

    private String name;
    private String description;
}

File: src/main/kotlin/com/leijendary/spring/template/model/SampleTableTranslations.kt
package com.leijendary.spring.template.model

import com.leijendary.spring.template.core.model.LocaleModel
import javax.persistence.Embeddable

@Embeddable
data class SampleTableTranslations(
    var name: String,
    var description: String,
    override var language: String,
    override var ordinal: Int
) : LocaleModel(language, ordinal)